docs/fa/examples.md

```diff
@@ -0,0 +1,97 @@
+# Examples
+
+[FA Class](class-fa.md)  
+[NFA Class](class-nfa.md)  
+[DFA Class](class-dfa.md)  
+[Table of Contents](../README.md)
+
+On this page, we give some short examples with discussion for the finite automata classes and methods.
+
+## Reading basic input
+
+The following code snippet creates a `DFA` and prints whether it accepts or rejects user input.
+
+```python
+from automata.fa.dfa import DFA
+
+# DFA which matches all binary strings ending in an odd number of '1's
+my_dfa = DFA(
+    states={'q0', 'q1', 'q2'},
+    input_symbols={'0', '1'},
+    transitions={
+        'q0': {'0': 'q0', '1': 'q1'},
+        'q1': {'0': 'q0', '1': 'q2'},
+        'q2': {'0': 'q2', '1': 'q1'}
+    },
+    initial_state='q0',
+    final_states={'q1'}
+)
+
+try:
+    while True:
+        if my_dfa.accepts_input(input('Please enter your input: ')):
+            print('Accepted')
+        else:
+            print('Rejected')
+except KeyboardInterrupt:
+    print('')
+```
+
+## Subset for NFAs
+
+The `NFA` does not have a built-in method for checking whether it is a subset of another `NFA`. However, this can be done using existing methods.
```

Comment:
@eliotwrobson This would obviously be part of a separate PR, but do you think it would be worth adding `issubset`, `__le__`, `issuperset`, etc. to the `NFA` class to create parity with `DFA`?

Reply:
So in general yes, but I would like to use an algorithm other than the code here. Basically, I think that every built-in method to these classes should be optimized beyond calling other methods that construct other automaton under the hood. For the subset algorithm, there are faster algorithms than the one used here (don't involve constructing a separate NFA), see http://www.lsv.fr/~doyen/papers/Antichains_Algorithms_Finite_Automata.pdf for an algorithm that's supposed to be faster.

However, I think some of these algorithms are tricky to implement and would take some care by someone that knows what they're doing following a paper.

Reply:
@eliotwrobson All good points. Let's not worry about it then, at least for now (certainly not before the eventual v8 release).

Reply:
@caleb531 Sounds good! I'll go ahead and resolve this discussion, and since this is the last item, I'll merge in just a sec.